src/wrappers/any.jl

```diff
@@ -165,7 +165,11 @@ function start_worker(raylet_socket, store_socket, ray_address, node_ip_address,
                                           # ```
                                           # Using `ConstCxxRef` doesn't seem supported
                                           # (i.e. `const &`)
-                                          (RayFunctionAllocated,))
+                                          (RayFunctionAllocated,
+                                           # TODO: can simplify this I think?
```

Comment:
we could make a type alias if we want?

Reply:
eh, don't think it's really worth the added indirection if the type has to be fully concrete anyway...